WalkingChallenge.xcodeproj/project.pbxproj

```diff
@@ -630,10 +630,10 @@
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				CLANG_ENABLE_MODULES = YES;
 				CODE_SIGN_ENTITLEMENTS = WalkingChallenge/WalkingChallenge.entitlements;
-				DEVELOPMENT_TEAM = 6SQ8F32998;
+				DEVELOPMENT_TEAM = 45YATE627L;
 				INFOPLIST_FILE = WalkingChallenge/Info.plist;
 				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/Frameworks";
-				PRODUCT_BUNDLE_IDENTIFIER = com.akf.WalkingChallenge;
+				PRODUCT_BUNDLE_IDENTIFIER = com.akf.WalkingChalleng;
```

Comment:
Why this change?

Reply:
App won't run on my device because of a provisional profile issue - this was my workaround. This should've been changed back before committing.